frontend/core-ui/src/modules/automations/utils/automationConnectionUtils.ts

```
@@ -152,3 +159,89 @@ export const getNewId = (checkIds: string[]) => {
+  nodes,
+  triggers,
+  actions,
+}: {
+  edge: EdgeProps;
+  setEdges: Dispatch<SetStateAction<Edge<EdgeProps>[]>>;
+  nodes: Node<NodeData>[];
+  triggers: ITrigger[];
+  actions: IAction[];
+}) => {
+  setEdges((eds: Edge<EdgeProps>[]) => eds.filter((e) => e.id !== edge.id));
+  let info: any = { source: edge.source, target: undefined };
```

ESLint

[error] 'info' is never reassigned. Use 'const' instead.

(prefer-const)